src/bfbs_gen_lua.cpp

```diff
@@ -107,7 +133,7 @@ class LuaBfbsGenerator : public BaseBfbsGenerator {
 
       std::string ns;
       const std::string object_name =
-          NormalizeName(Denamespace(object->name(), ns));
+          namer_.ObjectType(Denamespace(object->name(), ns));
```

Comment:
I think you mean `namer_.Type` because the symbol is for a type in Lua. As best I can tell this code generates the normal struct/table, not the native version, and I don't think Lua supports the object api... does it?

`namer_.ObjectType` was so named because it was for the "object based API", i.e. the API which copies data into a relatively native/ergonomic structure, that's generated only if we pass `--gen-object-api`. Typically, `ObjectType` returns the same string as `Type` but with a prefix and suffix. You set those to "" in `LuaDefaultConfig`, so `Type` and `ObjectType` return the same thing for now, but I think you still mean `Type`.

We can rename `Namer::ObjectType` into `Namer::ObjectApiType`  if that's clearer

It's really unfortunate that we have `Namer::ObjectType`, `r::Object`, and "Object API"... words are hard 😢

Reply:
Done.

I like the renaming to ObjectAPIType suggestion.